### ISSUE TITLE
More weights and --visualize bug

### ISSUE THREAD

**Issue by chevalchen** [ClaudiaCuttano/SANSA#7]
Hi，thank you for opening source such an excellent job!
By the way, could you kindly share the SANSA model weights trained that were trained on COCO and ADE20k and +LVIS in the Generalist In-context models experiment？We’d truly appreciate your support!  
And i also find a bug on --visualize
Bug: --visualize flag causes error in inference_fss.py due to incorrect iou argument type
Description:
When running inference with the --visualize flag, the original code passes a tensor to the iou parameter of visualize_episode:
# inference_fss.py (original)
visualize_episode(
    ...
    iou=area_inter/area_union,  # Bug: Tensor [2, B], not a scalar
)
area_inter and area_union are both tensors of shape [2, B] (rows = background/foreground channels, columns = batch samples). Dividing them element-wise produces another tensor rather than a scalar float, which causes a runtime error when visualize_episode attempts to format the IoU value into the figure title.
Fix:
Extract the foreground IoU as a scalar before passing it to the function:
fg_inter = area_inter[1].sum().item()
fg_union = area_union[1].sum().item()
vis_iou = fg_inter / max(fg_union, 1e-6)

visualize_episode(
    ...
    iou=vis_iou,  # scalar float
)
Additionally, visualize_episode in util/visualization.py should include a defensive _to_scalar() helper to gracefully handle cases where a tensor is passed:
def _to_scalar(value):
    if isinstance(value, torch.Tensor):
        if value.numel() == 1:
            return float(value.item())
        value = value.detach().float().mean().item()
    return float(value)
And use it in the title rendering line:

fig.suptitle(f"IoU: {_to_scalar(iou):.2f}", ...)
Environment: Reproduced on the released checkpoint with --visualize --prompt mask
